config.cpp

```diff
@@ -46,6 +47,10 @@ bool loadConfig() {
             chipID = createCharString(banaConfig["chip_id"].as_string()->get());
         }
 
+        auto banaRedirConfig = config["bana_redir"];
+        if (banaRedirConfig["redirect"].as_boolean()->get())
```

Comment:
if theres no bana_redir in the config this will throw
check if bana_redir is a table first
```cpp
if (banaRedirConfig.is_table() && banaRedirConfig["redirect"].as_boolean()->get())
```

Reply:
Will do bossman